.github/find_reviewers.yml

```diff
@@ -8,22 +8,12 @@ labels:
     # example, when the label `ready-for-review` is applied, two reviewers are
     # chosen from the first group and one reviewer from the second group.
     groups:
-      -
-        number_of_picks: 2
-        possible_reviewers:
-          - florian
-          - erika
-          - octobot
-          - mathilda
-          - james
-          - cx-3po
       -
         number_of_picks: 1
         possible_reviewers:
-          - romain
-          - maria
-          - amal
-          - ana
+          - marianeum
+          - Crunch09
+          - jabeattie
     label: "documentation"
```

Comment:
Think we should update this from `documentation` as well and use something like `ready-for-review`